editor/src/messages/portfolio/document/node_graph/node_properties.rs

```diff
@@ -1903,44 +1833,37 @@ pub fn math_properties(node_id: NodeId, context: &mut NodePropertiesContext) ->
 }
 
 pub struct ParameterWidgetsInfo<'a> {
-	document_node: &'a DocumentNode,
+	document_node: Option<&'a DocumentNode>,
```

Comment:
Now an option so the error logic can be handled in the widget (once), rather than the user of the widget (multiple)